core/spec/models/spree/variant_spec.rb

```diff
@@ -159,7 +159,9 @@
       context "and a variant is really deleted" do
         let!(:old_option_values_variant_ids) { variant.option_values_variants.pluck(:id) }
 
-        before { variant.really_destroy! }
+        before do
+          Spree::Deprecation.silence { variant.really_destroy! }
```

Comment:
Would you mind to add a comment here to specify this will be replaced with `destroy` in solidus 3.0?